dist/index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.4.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+  <link rel="stylesheet" href="assets/css/style.css">
+  <title>Weather App</title>
+</head>
+
+<body>
+  <div class="full-cont">
+    <nav class="navbar navbar-dark bg-blue">
+      <a class="navbar-brand" href="#">
+        <img src="./assets/media/logo.png" width="30" height="30" class="d-inline-block align-top" alt="">
+        How is the Weather?
+      </a>
+
+      <form class="form-inline">
+        <input class="form-control mr-sm-2" type="search" placeholder="City" aria-label="Search" id="cityInput">
+        <button class="btn btn btn-light my-2 my-sm-0 mr-1" type="button" id="celsius">Celsius</button>
```

Comment:
Opps, I guess you miss some important information
 Your website looks really nice but we need to follow the necessary instructions given on `Odin` page as you know all Microverse students follow it.
so, we need to use `toggle` not as a form button`


You need to add toggle control to determine if the result will be shown in Celsius or Fahrenheit (According to the Odin instructions, screenshot below;)

![image](https://user-images.githubusercontent.com/31897434/69664063-e2356180-1087-11ea-8e7d-e94e88863eb0.png)

https://www.theodinproject.com/courses/javascript/lessons/weather-app

Reply:
Ok thanks i will fix it